src/omnicore/omnicore.cpp

```diff
@@ -3273,6 +3285,17 @@ void CMPTxList::recordMetaDExCancelTX(const uint256 &txidMaster, const uint256 &
        }
 }
 
+void CMPTxList::recordSendAllSubRecord(const uint256& txid, int subRecordNumber, uint32_t propertyId, int64_t nValue)
+{
+    if (!pdb) return;
+
+    const std::string& key = strprintf("%s-%d", txid.ToString(), subRecordNumber);
```

Comment:
Quick note @zathras-crypto: using `const T&` is great in a context, where the string [...] already exists, say when accessing a value in a map, but in this case simply using `[const] std::string key = strprintf(..);` is probably to prefer, because we truely want to create a new string, and not point to one. At least this is my understanding.